libraries/ESP8266HTTPClient/examples/DigestAuthorization/DigestAuthorization.ino

```diff
@@ -0,0 +1,105 @@
+/*
+
+   This example is released into public domain,
+   or, at your option, CC0 licensed.
+ */
+
+#include <ESP8266WiFi.h>
+
+#include <ESP8266HTTPClient.h>
+
+const char* ssid = "........";
+const char* ssidPassword = "........";
+
+const char *username = "admin";
+const char *password = "admin";
+
+const char *server = "http://httpbin.org";
+const char *uri = "/digest-auth/auth/admin/admin/MD5";
+
+String exractParam(String& authReq, const String& param, const char delimit){
+	int _begin = authReq.indexOf(param);
```

Comment:
The comment about reformatting the sketch using Arduino IDE is still valid, by the way. Arduino sketches use 2 spaces for indentation, this file uses tabs.